package_cellink_index.json

```diff
@@ -12,12 +12,12 @@
             {
                "name": "NeoCortex",
                "architecture": "avr",
-               "version": "1.0.0",
+               "version": "1.1.0",
                "category": "Contributed",
-               "url": "https://raw.githubusercontent.com/CELLINKAB/lowlevelboard-arduino-utils/master/neocortex-1.0.0.zip",
-               "archiveFileName": "neocortex-1.0.0.zip",
-               "checksum": "SHA-256:b93b0abb16515a9c0df408b2213ff854856c276689344331ce0cbd15e19ee916",
-               "size": "4155",
+               "url": "https://raw.githubusercontent.com/CELLINKAB/lowlevelboard-arduino-utils/neocortex2/neocortex-1.1.0.zip",

```

Comment:
Looks good! Let's change that back to `master` and merge :+1:

Reply:
thanks for reminding!